tap_pendo/streams.py

```diff
@@ -483,8 +483,12 @@ def send_request_get_results(self, req):
 
             resp.raise_for_status()
 
+            # Return list of records instead of yielding because more than one iteration occur over data in tap flow
+            # and yield will return generator which flushes out after one iteration.
+            to_return = []
             for item in ijson.items(resp.raw, 'results.item'):
-                yield humps.decamelize(item)
+                to_return.append(humps.decamelize(item))
+            return to_return
```

Comment:
@savan-chovatiya Did you confirm that there are no other parent-child streams having cases similar to visitors-visitor_history streams